docs/standard/library-guidance/strong-naming.md

```diff
@@ -24,17 +24,19 @@ The downside to strong naming is that .NET Framework on Windows enables strict l
 
 When .NET developers complain about strong naming, what they're usually complaining about is strict assembly loading. Fortunately, this issue is isolated to .NET Framework. .NET 5+, .NET Core, Xamarin, UWP, and most other .NET implementations don't have strict assembly loading, which is the main downside of strong naming.
 
-One important aspect of strong naming is that it's viral: a strong named assembly can only reference other strong named assemblies. If your library isn't strong named, then you have excluded developers who are building an application or library that needs strong naming from using it.
+One important aspect of strong naming on .NET Framework is that it's viral: a strong named assembly can only reference other strong named assemblies. If your library isn't strong named, then you have excluded developers who are building an application or library that needs strong naming from using it on .NET Framework.
 
-The benefits of strong naming are:
+The benefits of strong naming on .NET Framework are:
 
 1. The assembly can be referenced and used by other strong-named assemblies.
 2. The assembly can be stored in the Global Assembly Cache (GAC).
 3. The assembly can be loaded side by side with other versions of the assembly. Side-by-side assembly loading is commonly required by applications with plug-in architectures.
 
+Strong naming has no benefits on .NET Core/5+.
```

Comment:
No benefit, or also actually no effect?

Reply:
If you add/remove/change strong name keys on a library is that still a binary breaking change in .NET Core/5+? If it is, then strong naming still has an effect.

Reply:
> If you add/remove/change strong name keys on a library is that still a binary breaking change in .NET Core/5+? 

The runtime is going to be fine, but some of the build tools are likely going to choke on it. So yes - it is still a breaking change.